browser/ui/brave_wallet/BUILD.gn

```diff
@@ -1,4 +1,7 @@
-assert(!is_android)
```

Comment:
why do we need to modify that file? We don't use that include on Android, please revert.

Reply:
fixed